src/script/components/panel/SingleAction/SingleAction.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+
+import {Button, ButtonVariant, FlexBox} from '@wireapp/react-ui-kit';
+
+import {t} from 'Util/LocalizerUtil';
+
+export interface MenuItem {
+  click: () => void;
+  icon: string;
+  identifier: string;
+  label: string;
+}
+
+export interface SingleActionProps {
+  item: MenuItem;
+  onCancel: (action: any) => void;
```

Comment:
can't we be more specific with the action type?

Reply:
I think it doesn't even need arguments, isn't it @V-Gira ?

Reply:
so we pass `() => void` to UserActions [here](https://github.com/wireapp/wire-webapp/blob/virgile/user-actions-modal/src/script/components/Modals/UserModal/UserModal.tsx#L221) from `UserModal`
Then `onAction` gets typed to `(actions: Actions) => void` [here](https://github.com/wireapp/wire-webapp/pull/16171/files#diff-f8053f2d1c4626a4daa1de18c4b8a9c3c9ba1fafa766a2de19d7cf26ad9e358eR75)

If I remove the argument from `SingleAction.tsx` it would complain that I can't pass `(actions: Actions) => void`  to `() => void` :upside_down_face: 

I could add `do-nothing` as an action type or pass `{onAction as any}` [here](https://github.com/wireapp/wire-webapp/pull/16171/files#diff-f8053f2d1c4626a4daa1de18c4b8a9c3c9ba1fafa766a2de19d7cf26ad9e358eR317) instead